tasks/bump.js

```diff
@@ -143,12 +143,21 @@ module.exports = function(grunt) {
     runIf(opts.commit, function() {
       var commitMessage = opts.commitMessage.replace('%VERSION%', globalVersion);
 
-      exec('git commit ' + opts.commitFiles.join(' ') + ' -m "' + commitMessage + '"', function(err, stdout, stderr) {
+      var force = opts.commitForceAdd ? ' -f' : '';
+
+      exec('git add ' + opts.commitFiles.join(' ') + force, function(err, stdout, stderr) {
```

Comment:
i'd do an `if (force)` to use the new logic and the else to keep the same logic if force isn't used. Although doing that might mean the requirement of adding a promise library or moving the commit to another function that can be referenced from the add callback or called in the else